tests/phpunit/integration/Core/Authentication/Google_ProxyTest.php

```diff
@@ -26,11 +26,12 @@ public function test_get_site_fields() {
 
 		$this->assertEqualSetsWithIndex(
 			array(
-				'url'        => home_url(),
-				'action_uri' => admin_url( 'index.php' ),
-				'name'       => get_bloginfo( 'name' ),
-				'return_uri' => $context->admin_url( 'splash' ),
-				'admin_root' => parse_url( admin_url( '/' ), PHP_URL_PATH ),
+				'url'                    => home_url(),
+				'action_uri'             => admin_url( 'index.php' ),
+				'name'                   => get_bloginfo( 'name' ),
+				'return_uri'             => $context->admin_url( 'splash' ),
+				'admin_root'             => parse_url( admin_url( '/' ), PHP_URL_PATH ),
+				'analytics_redirect_uri' => add_query_arg( 'gatoscallback', 1, home_url() ),
```

Comment:
This test should probably use untrailingslashit on the home url, like the function it is testing, right? will this pass if the test site home URL has a trailing slash?

Reply:
Thanks for catching that @adamsilverstein! I've updated the test.